ReactSkia/sdk/NotificationCenter.h

```diff
@@ -113,22 +118,22 @@ unsigned int NotificationCenter::on(std::string eventName, std::function<void (A
 
 template <typename... Args>
 void NotificationCenter::emit(std::string eventName, Args... args) {
-    std::list<std::shared_ptr<Listener<Args...>>> handlers;
-    
-    {
-        std::lock_guard<std::mutex> lock(mutex);
+    //Creating the dispatch Handler for dispach the event in the event base folly thread. 
+    auto dispatchHandler = [=](){   
```

Comment:
This lambda will be created every time when we emit a signal . I think we can make it static or private member so that it will be created only once.

Reply:
it difficult to keep it as private member function/static member function, because.
1. Every time we change the event name. Since it is lambda with copy(in capture block) it can hold the eventname as local reference.
2. Hard to bind the variable number of arguments to thread function.